src/OpenTelemetry/CHANGELOG.md

```diff
@@ -12,6 +12,11 @@
   function when configuring a view (applies to individual metrics).
   ([#5542](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5542))
 
+* **Experimental (pre-release builds only):** The default `ExemplarFilterType`
+  on `MeterProvider` is now `ExemplarFilterType.TraceBased` which will enable
```

Comment:
What's the perf implications that the users should be aware of?

Reply:
Let's wait on this. My guess is most users will skip over anything prefixed with `**Experimental (pre-release builds only):**` in the CHANGELOG. What I think would be more useful is on the final entry where we make everything public for stable builds we can add a link there to something in the docs. Thinking like: `Understanding performance implications when sampling Exemplars`.

Reply:
I'm fine to address the changelog/doc later.

I think we still need to know the perf implication of this PR as it serves as a critical input while making decisions.

Reply:
Updated the description with some benchmarks.

Reply:
Could you also provide the increase in memory consumption per metric since we would now allocate an `ExemplarReservoir` instance for each MetricPoint?